test/logflare/logs/logs_test.exs

```diff
@@ -114,6 +99,7 @@ defmodule Logflare.LogsTest do
         assert conn.adapter == nil
         schema = body.schema
         assert %_{name: "key", type: "STRING"} = TestUtils.get_bq_field_schema(schema, "key")
+        send(pid, :ok)
```

Comment:
I would add some context to this message, to make it clearer that it comes from this callback.